pkg/services/ngalert/notifier/alertmanager.go

```diff
@@ -27,8 +27,8 @@ import (
 )
 
 const (
-	notificationLogFilename = "notifications"
-	silencesFilename        = "silences"
+	NotificationLogFilename = "notifications"
+	SilencesFilename        = "silences"
```

Comment:
We need to export these constants to use them in the `remote` package.
All further changes in this file are just capitalizations of these two constant names. The same goes for `multiorg_alertmanager.go` and `multiorg_alertmanager_test.go`